### DISCUSSION TITLE
无法打开B站客服

### DISCUSSION THREAD

**Discussion by ManyullynSword** [ywmoyue/biliuwp-lite#791] [bug]
### Bug 描述

我遇到了一些问题，需要联系B站客服，在线客服选择了使用哔哩哔哩UWP打开，但是哔哩哔哩UWP无法打开客服界面。

### 操作步骤

```plain text
1.打开https://www.bilibili.com/h5/faq
2.点击在线客服
3.选择使用哔哩哔哩UWP打开
4.打开了哔哩哔哩UWP的主页，但是无法进入客服页面，再点击超链接多少次都没用。
```

### 期望结果

能够正常打开客服界面

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build不知道是19041还是19045)

### 设备类型

台式机

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
这是因为点击在线客服后会调用一个`bilibili://`协议的链接，这个链接会唤醒对应的app打开对应的页面

但bililite只适配了这个协议的视频链接，没有适配也没做在线客服的链接和页面

从这个网页的样式来看，我猜是给移动端安卓或ios的官方app使用的
